.changeset/lovely-stingrays-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/account": patch
+"fuels": patch
+---
+
+chore: optimize `block` gql queries
```

Comment:
Can you mark this as breaking and describe what those are in the PR description?